apidoc/Titanium/UI/ListView.yml

```diff
@@ -902,6 +902,15 @@ properties:
     platforms: [android, iphone, ipad, macos]
     since: 3.2.0
 
+  - name: continuousUpdate
+    summary: Determines if the scrolling event should fire every time there is a new visible item.
+    description: |
+        When setting `continuousUpdate` to `true` the [scrolling](Titanium.UI.ListView.scrolling) event will everytime a new item is visible.
```

Comment:
Would you mind documenting what the use-case for enabling this is in the description please?
I'm curious why you want this feature. And I'm willing to help add it to iOS.

Also, the following sentence is awkward.
`...event will everytime a new item is visible.`
I think the following is better. _(Not trying to be picky.)_
`...event will fire every time a new item is scrolled in.`

Reply:
Sure, will update the docs tomorrow. I need to know where I'm at to change a menu e.g. which element just came into view. Currently I'm using markers but they only fire once so I have to add them again when I've hit one. 
Especially iOS had problems when I've add the markers and it filled the list after adding them. It did fire the marker event for all markers already because it was triggered before the actual data was visible and my 2nd (normally offscreen marker) was visible.
Having the scoll event just telling me the whole time which element is visible (fistVisibleItem/section) I can easily use that for lazy loading, dynamically switching menus (indicator where you are) and other triggers (e.g. changing a headline)